presto-docs/src/main/sphinx/presto_cpp/features.rst

```diff
@@ -184,6 +184,17 @@ disabled if ``connector.num-io-threads-hw-multiplier`` is set to zero.
 
 Whether async data cache is enabled.
 
+``async-cache-ssd-by-default-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type** ``bool``
+* **Default value:** ``true``
+
+If ``true``, SSD cache is enabled by default and is disabled only if
```

Comment:
I would leave out the implementation logic and only give high level introduction